jupyterlab_gitplus/utils.py

```diff
@@ -2,8 +2,11 @@
 import re
 import os
 import stat
+import logging
+_logger = logging.getLogger(__name__)
```

Comment:
Let's just call it `logger` (instead of `_logger`) to be consistent with other log statements in the repo.

Reply:
Done